tapchannel/aux_sweeper.go

```diff
@@ -569,10 +572,16 @@ func (a *AuxSweeper) createAndSignSweepVpackets(
 		return lfn.Ok(vPkts)
 	}
 
-	return lfn.AndThen2(
+	return lfn.FlatMapResult(
```

Comment:
nice 🙏  glad we didnt need to do `AndThen(AndThen())`

Reply:
We kind of had to... Just that `AndThen` is now called `FlatMapResult` :sweat_smile: